inc/mailcollector.class.php

```diff
@@ -970,7 +975,7 @@ function buildTicket($uid, \Zend\Mail\Storage\Message $message, $options = []) {
          foreach ($tos as $to) {
             if ($to != $requester
                && !Toolbox::inArrayCaseCompare($to, $blacklisted_emails) // not blacklisted emails
-               && ($tmp = User::getOrImportByEmail($to) > 0)) {
+               && (($tmp = User::getOrImportByEmail($to)) > 0)) {
```

Comment:
```suggestion
               && ($tmp = User::getOrImportByEmail($to)) > 0) {
```

Reply:
revert to previous check version

Reply:
this ```&& ($tmp = User::getOrImportByEmail($to) > 0)``` affected ```true``` value to ```$tmp``` variable while 

 ```&& (($tmp = User::getOrImportByEmail($to)) > 0))``` affected users_id value to ```$tmp``` variable

Reply:
At this point collector need users_id rather than succes (or not) when trying retrieve user